asn1/InterledgerProtocol.asn

```diff
@@ -40,4 +40,9 @@ InterledgerProtocolError ::= SEQUENCE {
     data OCTET STRING (SIZE (0..8192))
 }
 
+InterledgerSideProtocolData ::= SEQUENCE OF SEQUENCE {
+  protocolName IA5String,
+  protocolData OCTET STRING
+}
```

Comment:
The way this is defined, you can't actually send side protocol data along with each of the standard messages. This is really just a way to send a custom message in one of the requests. I think it's useful to allow each message to be extended, in addition to allowing fully custom messages to be sent via this protocol.

Also, if one of these custom things is going to become part of the standard packet set I would rather make it a single set of `protocolName` (maybe `mimeType`) and `protocolData`, instead of the `SEQUENCE OF` that.